apps/examples/elf/micomapp/Makefile

```diff
@@ -113,6 +113,10 @@ clean:
 install:
 	$(Q) mkdir -p $(USER_BIN_DIR)
 	$(Q) install $(BIN) $(USER_BIN_DIR)/$(BIN)
+ifeq ($(CONFIG_ELF_EXCLUDE_SYMBOLS),y)
```

Comment:
I suggest two location to add this.
1. Makefile.unix
```
...
ifeq ($(CONFIG_RAW_BINARY),y)
	$(Q) echo "CP: $(BIN_EXE).bin"
	$(Q) $(OBJCOPY) $(OBJCOPYARGS) -O binary $(BIN) $(BIN).bin
endif
HERE!!!!!!!!!!!!!!!!!!!!!!!!!
	$(Q) echo "Start the Board Specific Work for Binary"
	$(Q) $(call MAKE_BOARD_SPECIFIC_BIN, $(BIN_EXE), $(BIN_EXT))
...
```
2. In MAKE_BOARD_SPECIFIC_BIN definition of each board.

Reply:
We have below line in the micomapp/Makefile and wifiapp/Makefile

`$(Q) $(TOPDIR)/tools/mkbinheader.py $(USER_BIN_DIR)/$(BIN) $(BIN_TYPE) $(KERNEL_VER) $(BIN) $(BIN_VER) $(DYNAMIC_RAM_SIZE)`

It is not possible to apply the strip command after adding the header. So, I need to make these changes in these Makefile's only.